census/anaconda-project.yml

```
@@ -1,45 +1,34 @@
-
-
 # To reproduce: install 'anaconda-project', then 'anaconda-project run'
 name: census
 description: Visualize 2010 Census demographic data
 examples_config:
+  created: 2016-03-27
   maintainers:
   - jbednar
   labels:
   - datashader
-  - geoviews
 
 user_fields: [examples_config]
 
-channels: [pyviz,conda-forge,defaults]
+channels: [conda-forge]
```

When conda-forge is used, I suggest adding `nodefaults` to let anaconda-project know that it should exclude `defaults`:

```suggestion
channels: [conda-forge, nodefaults]
```